app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt

```diff
@@ -164,7 +165,7 @@ fun HomeScreen(
             }
           },
           bottomBar = { NavigationBar(navigation = navigation) },
-          modifier = Modifier.fillMaxWidth().testTag("HomeScreen")) { innerPadding ->
+          modifier = Modifier.fillMaxWidth().testTag("HomeScreen")) {
```

Comment:
wanted to remove this parameter for a while but always got an error aha. Thank for making it work!